PerfView.sln

```diff
@@ -76,89 +76,249 @@ EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
+		Debug|x64 = Debug|x64
+		Debug|x86 = Debug|x86
```

Comment:
Do you know why VS added these configurations?   I don't much care, as they are benign, but I am surprised your change triggered it.

Reply:
I don't offhand.  I did not interact with the configuration manager UI or change the current config but VS inserted the entries none-the-less.